.github/workflows/test-functional.yml

```diff
@@ -34,7 +34,11 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
       - name: echo github context
         if: always()
-        run: echo "${{ github.event.workflow_run.head_repository.full_name }}-${{ github.event.workflow_run.head_branch }}-${{ github.workflow_ref }}"
+        env:
+          FULL_NAME: ${{ github.event.workflow_run.head_repository.full_name }}
+          HEAD_BRANCH: ${{ github.event.workflow_run.head_branch }}
+          WORKFLOW_REF: ${{ github.workflow_ref }}
+        run: echo "$FULL_NAME-$HEAD_BRANCH-$WORKFLOW_REF"
 
```

Comment:
We can actually just delete the offending `echo`, it is unused and a silly leftover from testing.